articles/cosmos-db/create-mongodb-golang.md

```diff
@@ -0,0 +1,242 @@
+
+3. In the **New account** blade, specify the desired configuration for the Azure
+Cosmos DB account.
+
+With Azure Cosmos DB, you can choose one of four programming models: Gremlin
+(graph), MongoDB, SQL (DocumentDB), and Table (key-value).
+
+In this quick start we’ll be programming against the MongoDB API so you’ll
+choose **MongoDB** as you fill out the form. But if you have graph data for a
+social media mes New app, document data from a catalog app, or key/value (table)
+data, realize that Azure Cosmos DB can provide a highly available,
+globally-distributed database service platform for all your mission-critical
+applications.
+
+Fill out the New account blade using the information in the screenshot as a
+guide . You will choose unique values as you set up your account so your values
+will not match the screenshot exactly
+
+![](https://cdn-images-1.medium.com/max/800/1*XdFlpeNKLe7o3FOtCZ17AQ.png)
+<span class="figcaption_hack">Create Azure Cosmos DB Account</span>
+
+4. Click **Create** to create the account.
+
+5. On the toolbar, click **Notifications** to monitor the deployment process.
+
+![](https://cdn-images-1.medium.com/max/800/1*WTGVkPwkSTxjUzKm2u09Eg.png)
+<span class="figcaption_hack">Check Progress of Azure Cosmos DB Deployment</span>
+
+6. Click on **golang-couch** resources.
+
+7. Get connection string information which will be required by client
+applications.
+
+![](https://cdn-images-1.medium.com/max/800/1*XoslPMsWsv5vqK2vhYAJPw.png)
+<span class="figcaption_hack">Connection information of MongoDB</span>
+
+### Setting up your application
+
+It’s time to make our hands dirty. Open your favorite editor (Gogland, VS Code
+or Atom). For this article, I will use Gogland editor.
+
+1.  Create folder CosmosDBAcces folder inside GOROOT\src folder
+1.  Run below command to get mgo package
+
+    go get gopkg.in/mgo.v2
+
+### Connect to an Azure Cosmos DB account
+
+[mgo](http://labix.org/mgo) (pronounced as *mango*) is a
+[MongoDB](http://www.mongodb.org/) driver for the [Go
+language](http://golang.org/) that implements a rich and well tested selection
+of features under a very simple API following standard Go idioms.
+
+Azure Cosmos DB supports the SSL-enabled MongoDB. To connect to an SSL-enabled
+MongoDB, we need to define the **DialServer** function in
+[mgo.DialInfo](http://gopkg.in/mgo.v2#DialInfo), and make use of the
+[tls.*Dial*](http://golang.org/pkg/crypto/tls#Dial) to perform the connection.
+
+The following code Golang code snippet connects Go app with Azure  MongoDB. 
+*DialInfo* holds options for establishing a session with a MongoDB cluster.
+
+```go
+// DialInfo holds options for establishing a session with a MongoDB cluster.
+dialInfo := &mgo.DialInfo{
+    Addrs:    []string{"golang-couch.documents.azure.com:10255"}, // Get HOST + PORT
+    Timeout:  60 * time.Second,
+    Database: "golang-coach",                                                                             // It can be anything
```

Comment:
Move comment close to ',' so that the entire line can fit into a single viewport.